pointmatcher.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=@CMAKE_INSTALL_PREFIX@
+libdir=${prefix}/lib
+includedir=${prefix}/include
+
+Name: pointmatcher 
+Description: See https://github.com/ethz-asl/libpointmatcher 
+Version: @PROJECT_VERSION@
+Libs: -L${libdir} -lpointmatcher
+Cflags: -I${includedir}/pointmatcher 
```

Comment:
```pointmatcher``` is too much here. Client code should include libpointmatcher header files with something like ```#include <pointmatcher/WHATEVER_FILE.h>```. But see also my upcoming more general comment.